index.html

```diff
@@ -4292,6 +4306,454 @@ <h3>Proof Formats</h3>
+
+      <p>
+This section contains algorithms that can be used by implementations to
+perform common operations.
+      </p>
+
+      <p class="note" title="Implementers can include additional checks, warnings, and errors.">
+Implementers are advised that the algorithms in this section contain the bare
+minimum set of checks used by implementations to test conformance to this
+specification. Additional checks that report helpful warnings for developers to
+help debug potential issues are expected to be provided by implementations.
+Similarly, additional checks that could result in new types of errors being
+reported in order to stop harmful content are also expected to be provided by
+implementations. Any of these additional checks might be integrated into future
+versions of this specification.
+      </p>
+
+      <section class="normative">
+        <h3>Verification</h3>
+
+        <p>
+This section contains an algorithm that <a>conforming verifier
+implementations</a> can run when verifying a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
+<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
+inputs, and produces an object (<var>verified</var>) that contains the
+following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>verified</var> to an empty object.
+          </li>
+          <li>
+Ensure that the securing mechanism has properly protected the
+<var>conforming document</var> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>securingMechanismVerified</var> to the result of executing the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> with <var>inputBytes</var> and <var>mediaType</var> passed as
+inputs to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>mediaType</var>,
+<var>warnings</var>, and <var>errors</var> in <var>verified</var> to their
+respective properties in <var>securingMechanismVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a document that cannot be considered to be
+secured. Return <var>verified</var>, which will contain warning and error
+information related to the verification failure.
```

Comment:
```suggestion
information related to the verification failure. At least one error MUST be included.
```

Reply:
This text is not necessary because the securing mechanism verification algorithm is guaranteed to set at least one error:

https://github.com/w3c/vc-data-model/pull/1338/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R4649-R4652